app/controllers/agent/RepresentationController.scala

```diff
@@ -71,6 +71,14 @@ class RepresentationController @Inject() (
       } yield Ok(confirmRevokeClientPropertyView(property.addressFull))
     }
 
+  // this endpoint only exists so we don't 404 when changing language after getting an error on submit
+  def showRevokeClientPropertyConfirmed(uarn: Long, plSubmissionId: String): Action[AnyContent] =
+    authenticated.async { implicit request =>
+      for {
+        property <- vmvConnector.getPropertyHistory(uarn)
```

Comment:
im guessing this isnt saved in a cache/session data anywhere is it? Annoying to have to do an extra vmv connector call just for reloading the page